packages/react/runtime/src/lynx-api.ts

```diff
@@ -87,19 +87,8 @@ export const root: Root = {
       __root.__jsx = jsx;
     } else {
       __root.__jsx = jsx;
-      let preactProcess: (() => void) | undefined = undefined;
-      // eslint-disable-next-line @typescript-eslint/unbound-method
-      const oldDebounceRendering = options.debounceRendering;
-      options.debounceRendering = (cb) => {
-        preactProcess = cb;
-      };
```

Comment:
Why do we remove this

Reply:
This is redundant logic if we already call process before hydrate.